IDEAS/LIDEAS/Components/BaseClasses/LinearAirModel.mo

```diff
@@ -0,0 +1,87 @@
+within IDEAS.LIDEAS.Components.BaseClasses;
+model LinearAirModel
+  extends IDEAS.Buildings.Components.ZoneAirModels.BaseClasses.PartialAirModel(
+    TAir(stateSelect=if sim.linearise then StateSelect.prefer else StateSelect.default),
+    useFluPor=not linearise and useFluPor2);
+  parameter Boolean useFluPor2 = true "Set to false to remove fluid ports, overwritten by sim.linearise";
```

Comment:
do we still need this functionality? Otherwise best to remove it and first merge #837